internal/pkg/fetch/fetch.go

```diff
@@ -36,8 +36,8 @@ import (
 	"github.com/bomctl/bomctl/internal/pkg/options"
 )
 
-func Fetch(sbomURL string, opts *options.FetchOptions) error {
-	document, err := fetchDocument(sbomURL, opts)
+func Fetch(sbomURL string, opts *options.FetchOptions, outputFile *os.File) error {
+	document, err := fetchDocument(sbomURL, opts, outputFile)
```

Comment:
I think this should remain how it was, referencing `opts.OutputFile` and without the unneeded `outputFile` param.